fastlane/Fastfile

```diff
@@ -243,6 +243,25 @@ platform :ios do
 
   desc "Move app to App Store Review"
   lane :production do
+    # Login to Spaceship
+    fastlane_require 'spaceship'
+    teamID = "368M544MTT"
```

Comment:
This is already defined in our `fastlane/Appfile` and we should not hard code this here. We should use [`CredentialsManager::AppfileConfig.try_fetch_value(:team_id)` per docs](https://docs.fastlane.tools/advanced/Appfile/)